addons/cookoff/CfgSFX.hpp

```diff
@@ -1,12 +1,26 @@
 
 class CfgSFX {
-    class GVAR(CookOff) {
-        name = QGVAR(cookoff);
+    class GVAR(CookOff_low) {
+        name = QGVAR(cookoff_low);
         // Index 4 is percentage chance to play, in theory high pressure is way more likely
-        variant0[] = {PATHTOF(sounds\cookoff_low_pressure.ogg),6,1,400,0.1,0,0,0};
-        variant1[] = {PATHTOF(sounds\cookoff_mid_pressure.ogg),6,1,400,0.25,0,0,0};
-        variant2[] = {PATHTOF(sounds\cookoff_high_pressure.ogg),6,1,400,0.65,0,0,0};
-        sounds[] = {"variant0","variant1","variant2"};
+        sound[] = {QPATHTOF(sounds\cookoff_low_pressure.ogg),6,1,400,1,0,0,0};
+        sounds[] = {"sound"};
+        titles[] = {};
+        empty[] = {"",0,0,0,0,0,0,0};
+    };
+    class GVAR(CookOff_mid) {
+        name = QGVAR(cookoff_mid);
+        // Index 4 is percentage chance to play, in theory high pressure is way more likely
+        sound[] = {QPATHTOF(sounds\cookoff_mid_pressure.ogg),6,1,400,1,0,0,0};
+        sounds[] = {"sound"};
+        titles[] = {};
+        empty[] = {"",0,0,0,0,0,0,0};
```

Comment:
Probably make sense to have these classes inherit and just change the `name` and `sound[]` properties